amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -2,23 +2,38 @@ import * as React from 'react';
 import moment from 'moment-timezone';
 
 import ColumnDescEditableText from 'components/TableDetail/ColumnDescEditableText';
+import { GlobalState } from 'ducks/rootReducer';
 import { logClick } from 'ducks/utilMethods';
+import { ToggleRequestAction } from 'ducks/notification/types';
+import { openRequestDescriptionDialog } from 'ducks/notification/reducer';
 import { TableColumn } from 'interfaces';
 
+import { Dropdown, MenuItem } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
 // TODO: Use css-modules instead of 'import'
 import './styles.scss';
 
-interface DetailListItemProps {
+interface StateFromProps {}
```

Comment:
nit: You might be able to delete this and use `null` at the bottom but I'm not 100% sure. Same thing for `OpenRequestDescription`

Reply:
nit: Can also directly use `{}` below as well.